submit_cgap/scripts/submit_ontology.py

```diff
@@ -27,11 +29,15 @@ def main(simulated_args_for_testing=None):
     parser.add_argument('--submission_protocol', '--submission-protocol', '-sp',
                         choices=SUBMISSION_PROTOCOLS, default=SubmissionProtocol.S3,
                         help=f"the submission protocol (default {SubmissionProtocol.S3!r})")
-
+    parser.add_argument('--verbose', default=False, action='store_true', help='Include verbose output.')
+    parser.add_argument('--debug', default=False, action='store_true', help='Include debug output.')
     args = parser.parse_args(args=simulated_args_for_testing)
 
     with script_catch_errors():
 
+        if not verify_ontology_file(args.ontology_filename):
+            return
```

Comment:
While it's possible to return False, you'd still end up wanting to raise an error here. Ultimately the problem is you want it to `exit(1)`, not `exit(0)`. To make this pretty, I suggest just calling without the conditional here and expecting that function to raise an error (particularly, `dcicutils.command_utils.ScriptFailure`) which will keep the stack trace from showing, which I think is why you're doing this contortion here.